schema.sql

```diff
@@ -13,4 +13,35 @@ CREATE TABLE animals (
 /* Add a column species of type string to animals table. */
 
 AlTER TABLE animals
-ADD COLUMN species char(100);
\ No newline at end of file
+ADD COLUMN species char(100);
+
+/* Create owners table */
+
+CREATE TABLE owners (
+    id int NOT NULL PRIMARY KEY, 
```

Comment:
[OPTIONAL]

- While trying to create a table, I'd suggest you add `GENERATED ALWAYS AS IDENTITY` to the `id` column. This will make sure that a unique value is gotten for each record/row. That way, you do not need to add `id` when you're populating your table with data. Imagine you had millions of rows 😁

Reply:
@DammyShittu  Thank you, for your valuable feedback I will definitely take it into consideration.

Reply:
You're welcome, @algerina 
I'm glad I could help.

Cheers 🎉